Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -6373,6 +6373,8 @@
 				RUN_CLANG_STATIC_ANALYZER = NO;
 				SWIFT_OBJC_BRIDGING_HEADER = "Wikipedia/Code/Wikipedia-Bridging-Header.h";
 				WMF_HOCKEYAPP_IDENTIFIER = 2295c3698bbd0b050f257772dd2bdbb2;
```

Comment:
So these values that are user defined settings are bogus? Or just the other ones need to be secure? Also... if they're being written to the plist, anyone can read them. Maybe we're talking about different config vars...